app/Http/Controllers/CreatePostingPeriod.php

```
@@ -0,0 +1,184 @@
+            throw new \Exception('Cannot create next month period - previous monthly periods are not properly closed');
+        }
+    }
+
+    /**
+     * Legacy method for backward compatibility
+     * Redirects to the main invoke method
+     */
+    public function CreatePostingPeriod(?Request $request = null): JsonResponse
+    {
+        return $this->__invoke($request);
+    }
```

PHPMD (2.15.0)

The method CreatePostingPeriod is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)